docs/user_guides/fs/storage_connector/usage.md

```diff
@@ -48,20 +48,48 @@ and users should pass a Spark data format to the `data_format` argument.
 === "PySpark"
     ```python
     # read data into dataframe using path 
-    df = connector.read(data_format='csv', path='path/to/object')   
+    df = connector.read(data_format='data_format', path='fileScheme://bucket/path/')   
```

Comment:
Are you sure it has to be the full path?

Reply:
yes, tested for GCS. For S3 also looking at the code it seems to need full path.